x-pack/plugins/cases/common/constants.ts

```diff
@@ -5,8 +5,7 @@
  * 2.0.
  */
 
-import { DEFAULT_MAX_SIGNALS } from '../../security_solution/common/constants';
-
+const DEFAULT_MAX_SIGNALS = 100;
```

Comment:
I think we should put a comment here and in security solution that there is a dependency between the two plugins and if changed in security solution it should change in cases also.